api/app/controllers/spree/api/v2/option_types_controller.rb

```diff
@@ -0,0 +1,51 @@
+module Spree
+  module Api
+    module V2
+      class OptionTypesController < Spree::Api::BaseController
+        def index
+          if params[:ids]
+            @option_types = Spree::OptionType.includes(:option_values).accessible_by(current_ability, :read).where(id: params[:ids].split(','))
```

Comment:
Line is too long. [143/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.